client/my-sites/domain-tip/index.jsx

```diff
@@ -40,8 +41,14 @@ const DomainTip = React.createClass( {
 		};
 	},
 
+	noticeShouldDisplay() {
+		//bypass some of the upgrade nudge display logic
+		return true;
```

Comment:
@mtias The display logic was close, but not quite right for the domain tip. Should I pull out a dumb component from the UpgradeNudge instead?